aws-test/tests/aws_backup_vault/test-hydrate-expected.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "name": "{{ output.id.value }}",
+    "policy": "<null>"
```

Comment:
Can you add tests for backup vault policy?
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/backup_vault_policy